js/authentication.md

```diff
@@ -899,9 +899,8 @@ After the federated login, you can retrieve related JWT token from the local cac
 import { Cache } from 'aws-amplify';
 
 // Run this after the sign-in
-Cache.getItem('federatedInfo').then(federatedInfo => {
-     const { token } = federatedInfo;
-});
+const federatedInfo = Cache.getItem('federatedInfo');
```

Comment:
In React Native the `Cache.getItem` is async function. We should add that in the doc.

Reply:
Added a separate sample for React native